### PR TITLE
Update files: Content review

### PR DESCRIPTION
**Pull request by hlucey**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -17,31 +17,31 @@
       "short_description": "Deploy watsonx.ai SaaS to automate the deployment and setup of the IBM watsonx platform.",
       "long_description": "The watsonX.ai SaaS Automation with Assistant and Governance easily deploys and embeds AI across your business, manages all data sources, and accelerates responsible AI workflows on one platform.",
       "provider_name": "FinOps Team.",
-      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/README.md",
+      "offering_docs_url": "https://test.cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-responsibilities",
```
Comment (hlucey) [inline ibm_catalog.json:20]: This link will have to be updated to https://cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-responsibilities when we push to production

File: ibm_catalog.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
               }
             ]
           },
-          "release_notes_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/releases",
+          "release_notes_url": "https://test.cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-release-notes",
```
Comment (hlucey) [inline ibm_catalog.json:68]: This link will need to be updated to https://cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-release-notes when we push to production.

File: ibm_catalog.json (hunk 1 above)
Comment (mkrudele) [reply ibm_catalog.json:20]: Why docs-draft? Shouldn't be `docs`?

File: ibm_catalog.json (hunk 2 above)
Comment (mkrudele) [reply ibm_catalog.json:68]: Why `docs-draft`? Shouldn't be `docs`?

File: ibm_catalog.json [hunk 3]
```diff
@@ -173,11 +173,11 @@
...
               "key": "watsonx_assistant_plan",
               "type": "string",
-              "default_value": "do not install",
+              "default_value": "Do not install",
               "options": [
                 {
-                  "displayname": "do not install",
-                  "value": "do not install"
+                  "displayname": "Do not install",
+                  "value": "Do not install"
```
Comment (mkrudele) [inline ibm_catalog.json:180]: I think value is referenced in the code. It should be `do not install`.  Right @andreainnocenti ?

**Review by mkrudele — CHANGES_REQUESTED** [review]

File: ibm_catalog.json (hunk 3 above)
Comment (andreainnocenti) [reply ibm_catalog.json:180]: right

Comment (hlucey) [reply ibm_catalog.json:180]: But is this visible to users? @mkrudele @andreainnocenti 
There's no way to at least style the string correctly with capitalization?

**Review by mkrudele — APPROVED** [review]

**Pull request merged**